src/util.js

```diff
@@ -23,8 +22,25 @@ export let roundRect = function(obj, precision) {
   return obj;
 };
 
-export let throttle = function(fn) {
-  let throttledFn = _.throttle(fn, cfg.throttle);
-  throttledFn.now = fn;
-  return throttledFn;
+export let maybeThrottle = function(fn) {
+  fn.maybeThrottle = true;
+  return fn;
+};
+
+export let deeply = function(fn) {
```

Comment:
Couple things:
1) There is no function `_.deeply`
2) `fn` is invoked for array of values, not a single plain object value. 

Something like this works for me:
```
let deeply = function(fn) {
  return function(obj, iteratee) {
    return _.mapValues(obj, function(v, _name) {
      return _.isPlainObject(v) ? deeply(fn)(v, iteratee) : fn(v, iteratee);
    });
  };
};
```